gemfire-core/src/main/java/com/gemstone/gemfire/internal/snappy/StoreCallbacks.java

```diff
@@ -41,6 +41,8 @@
 
   boolean isColumnTable(String qualifiedName);
 
+  boolean isInternalBatchTable(String tableName);
```

Comment:
isColumnTable() does the same.

Reply:
Right. I noticed that but did not go into implementation. May be it should be renamed something else, as column table I thought was a valid table type like Row Table....